.github/ISSUE_TEMPLATE/documentation_report.md

```diff
@@ -1,61 +1,34 @@
 ---
 name: 📝 Documentation Report
 about: Ask us about docs
-
 ---
+## Documentation report
+<!--- Verify first that your improvement is not already reported on GitHub -->
+<!--- Also test if the latest release and devel branch are affected too -->
+<!--- This form is processed by Ansibot (machine), complete *all* sections as described -->
 
-<!---
-Verify first that your issue/request is not already reported on GitHub.
-THIS FORM WILL BE READ BY A MACHINE, COMPLETE ALL SECTIONS AS DESCRIBED.
-Also test if the latest release, and devel branch are affected too.
-ALWAYS add information AFTER (OUTSIDE) these html comments.
-Otherwise it may end up being automatically closed by our bot. -->
-
-##### SUMMARY
-<!--- Explain the problem briefly -->
-
-##### ISSUE TYPE
- - Documentation Report
+### Summary
+<!--- Explain the problem briefly below -->
 
-##### COMPONENT NAME
-<!--- Insert, BELOW THIS COMMENT, the name of the module, plugin, task or feature.
-Do not include extra details here, e.g. "vyos_command" not "the network module vyos_command" or the full path-->
+### Component name
+<!--- Write the name of the module, plugin, task or feature below -->
 
-##### ANSIBLE VERSION
-<!--- Paste, BELOW THIS COMMENT, verbatim output from "ansible --version" between quotes below -->
-```
+### Ansible version
+<!--- Paste verbatim output from "ansible --version" between quotes -->
```

Comment:
the user's local ansible version is only relevant to a docs issue if they are reporting an issue with module docs - a link to the documentation in html form would be helpful here, or a description of a scenario if the person is reporting documentation we should have that we don't

Reply:
Right, I was thinking we may still ask for the version they are using, because documentation fixes may be related to clarifying behaviour that may already be changed/fixed. But I agree, that in a some cases this would be irrelevant.